src/controllers/pokestop.js

```diff
@@ -13,8 +14,8 @@ class Pokestop extends Controller {
 		let query = `
 		select humans.id, humans.name, humans.type, humans.latitude, humans.longitude, invasion.template, invasion.distance, invasion.clean, invasion.ping from invasion
 		join humans on humans.id = invasion.id
-		where humans.enabled = true and
-		(invasion.grunt_type='${data.gruntType}' or invasion.grunt_type = 'everything') and
+		where humans.enabled = 1 and
```

Comment:
true and 1 are the same

Reply:
Problems with sqlite using true as this is translated to an int field in the database rather than a boolean